oe_multilingual.module

```diff
@@ -81,3 +83,76 @@ function oe_multilingual_configurable_language_presave(EntityInterface $entity)
     }
   }
 }
+
+/**
+ * Implements hook_block_view_BASE_BLOCK_ID_alter() for the Page Header block.
+ */
+function oe_multilingual_block_view_oe_theme_helper_page_header_alter(array &$build, BlockPluginInterface $block) {
```

Comment:
This function should be on oe_theme, isn't it?

Reply:
I talk with @brummbar and he saw me that can be on both side. So this remark is not a blocker.
For me, I prefer to see this function on oe_theme repo.

Reply:
I am thinking a lot concerning this point. I am very confident that we should put this function on **oe_theme** because **oe_theme** requires - dev - **oe_multilingual** and finnaly **oe_theme** have the test to check this block.

Reply:
No. It doesn't matter that the theme is using multilingual as a dev dependency.

* The theme provides some functionality: the header block. 
* The multilingual extends that functionality with some extra links in there. That is why hooks are created.

@imanoleguskiza Please use the alter approach.